packages/tupaia-web/src/features/Map/utils/getSnakeCase.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+export const getSnakeCase = (value?: string) => {
```

Comment:
I think there is actually a function in the `utils` package called `snakeKeys` which might work for this, although it takes an object, not a single value. Alternatively we have the `snake` package in the monorepo you can use as well.